lib/cell.rb

```diff
@@ -0,0 +1,31 @@
+class Cell
+  attr_accessor :value, :neighbours, :solutions
+  CELL_VALUES = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+  def initialize(value)
+    raise "Values must be less than 10!" unless value.length == 1
```

Comment:
- Raising in the constructor will still create the object, but in an invalid state (usually `nil`, which could cause errors later on)
- Since you're converting `#to_i` here, why not just pass in a number?